src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -136,8 +136,9 @@ kibana_vars=(
     xpack.reporting.queue.timeout
     xpack.reporting.roles.allow
     xpack.searchprofiler.enabled
-    xpack.security.authProviders
     xpack.security.authc.providers
+    xpack.security.authc.oidc.realm
```

Comment:
It feels like these should be added for 7.x, do we want these to be part of this PR which will only be merged to master for 8.0?

Reply:
I have no preference, I can either keep these changes (two whitelisted realm-related settings) here and create a 7.x-only PR just for these changes or I can move these changes to a dedicated PR that I'll merge to master and backport to 7.x (maybe even 7.2 if we want). What would you prefer?

P.S.: I guess a large chunk of the Kibana settings isn't whitelisted here and people just mount their `config.yml` to the container to set those.

Reply:
Let's
> move these changes to a dedicated PR that I'll merge to master and backport to 7.x

> P.S.: I guess a large chunk of the Kibana settings isn't whitelisted here and people just mount their config.yml to the container to set those.

Yeah, I'm not quite sure the history of this. It seems like if we want these settings to be used by Cloud, we'd want these to be settable here. This is based on the assumption that Cloud is using these settings, which they might not be.

Reply:
Got it, I had an impression that cloud doesn't use our docker builds and rather relies on https://github.com/elastic/cloud/blob/master/scala-services/adminconsole/src/main/scala/no/found/adminconsole/clusters/kibana/validation/KibanaUserSettingsValidator.scala, but let me double check with the Cloud team.

Reply:
Ah, that's entirely possible.

Reply:
Confirmed, both ECE and Cloud use their different whitelist/blacklist mechanisms for Kibana settings.

Reply:
> Let's
>  move these changes to a dedicated PR that I'll merge to master and backport to 7.x

Moved to https://github.com/elastic/kibana/pull/38861